### PR TITLE
[Feature] Updates admin + user views to match the new DynamoDB structure

### PR DESCRIPTION
**Pull request by Sqyphen**
## Description

Updates the communication between backend and DynamoDB to use to the new and improved Debate structure layout

## Implementation

^ read above (plz)

## Screenshot

n/a

## .env changes

n/a

## Additional requirements

Run `npm install` again as the uuid package has been added

[insert funny GIF here]

### CODE REVIEW

File: modules/structured.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ function display(req, res, data) {
   let commentsAgainst = [];
   const debate = data.debate;
 
+  console.log(debate);
```
Comment (jamiebrown201) [inline modules/structured.js]: Should we keep this in?

File: routes/router.js [hunk 2]
```diff
@@ -35,6 +35,8 @@ router.get("/type/:debateType", async (req, res) => {
   try {
     const { debateType } = req.params;
     let debateList = await dynamo_db.getDebateList(debateType);
+
+    console.log(debateList);
```
Comment (jamiebrown201) [inline routes/router.js]: And this

File: modules/structured.js (hunk 1 above)
Comment (Sqyphen) [reply modules/structured.js]: Nope, removed now :|

File: routes/router.js (hunk 2 above)
Comment (Sqyphen) [reply routes/router.js]: Nope, removed now :|

**Pull request merged**